worlds/pokemon_crystal/data/phone_data.yaml

```
@@ -1058,3 +1058,35 @@ correct_tracker:
       He says good
       luck figuring
       your way out.
+
+rival_uwu:
+  caller: withheld
+  script:
+    - |
+      Hewwo <player>,
+      hmph! it’s me,
+      youw supewiow!
```

```suggestion
      Hewwo <player>,
      it's <rival>-chan,
      youw supewiow!
```

He should definitely introduce himself

---

It's also gonna be ???-chan a lot of the time which is even funnier